### ISSUE TITLE
Augmenter la taille du graphique de l'occupation des lits de réa en Martinique

### ISSUE THREAD

**Issue by Ryse93** [rozierguillaume/covid-19#36]
L'occupation des lits de réa en Martinique a dépasser les 180% et du coup le graphique dépasse.

![saturation_rea_journ_Martinique](https://user-images.githubusercontent.com/69793084/128332913-a8a0dcb2-80f7-4aeb-aba7-f4a43f44bb8a.jpg)

**Comment by Assitan** [comment]
ça été fait non ? là l'echelle va jusqu'à 200

**Comment by Ryse93** [comment]
> ça été fait non ? là l'echelle va jusqu'à 200

Yes j’ai vue, mais dans le graphique le texte en rouge est toujours coupé.

**Comment by Assitan** [comment]
> > ça été fait non ? là l'echelle va jusqu'à 200
> 
> Yes j’ai vu, mais dans le graphique le texte en rouge est toujours coupé.

On doit encore augmenter, la réa est à 235 % ...

**Comment by Ryse93** [comment]
![](https://raw.githubusercontent.com/rozierguillaume/covid-19/master/images/charts/france/departements_dashboards/saturation_rea_journ_Martinique.jpeg)


Ah enfaite, le graphique change de taille tout seul mais le texte de rouge est toujours coupé, du coup faudrait prévoir un peu de marge.
